connexion/security.py

```diff
@@ -25,6 +59,21 @@
 logger = logging.getLogger("connexion.api.security")
 
 
+@dataclass
```

Comment:
Not used currently, was exploring the option to create classes for the security schemes instead of regular dicts.